### PR TITLE
Prep for v1.19.0

### PR DESCRIPTION
**Pull request by odow**
## Basic

 - [x] `version` field of `Project.toml` has been updated
       - If a breaking change, increment the MAJOR field and reset others to 0
       - If adding new features, increment the MINOR field and reset PATCH to 0
       - If adding bug fixes or documentation changes, increment the PATCH field

## Documentation

 - [x] Add a new entry to `docs/src/changelog.md`, following existing style

## Tests

 - [x] The `solver-tests.yml` GitHub action does not have unexpected failures.
       https://github.com/jump-dev/MathOptInterface.jl/actions/runs/5399018088

### CODE REVIEW

**Comment by odow** [comment]
This release is mostly so that we can release AmplNLWriter with ScalarNonlinearFunction support.

**Comment by odow** [comment]
- [ ] Wait for a resolution to https://github.com/jump-dev/MathOptInterface.jl/pull/2235

File: docs/src/changelog.md [hunk 1]
```diff
@@ -7,6 +7,31 @@ CurrentModule = MathOptInterface
...
+## v1.19.0 (August 3, 2023)
+
+### Added
+
+ - Added [`Utilities.eval_variables`](@ref) support for
+   [`ScalarNonlinearFunction`](@ref) (#2218) (#2219)
+ - Added support for [`ScalarNonlinearFunction`](@ref) in `FileFormats.NL`
+   (#2228) (#2231)
+ - Added [`Bridges.Constraint.ScalarQuadraticToScalarNonlinearBridge`](@ref) (#2233)
+ - Added [`Bridges.Constraint.IntegerToZeroOne`](@ref) (#2205)
```
Comment (odow) [inline docs/src/changelog.md:19]:
```suggestion
 - Added [`Bridges.Constraint.IntegerToZeroOneBridge`](@ref) (#2205)
```

File: docs/src/changelog.md [hunk 2]
```diff
@@ -7,6 +7,31 @@ CurrentModule = MathOptInterface
...
+
+### Fixed
+
+ - Fixed method ambiguities in `operate` (#2224)
+ - Fixed reading LP file with a `-infinity <= x <= +infinity` variable (#2225)
+ - Fixed missing `require` in `Test.test_nonlinear_duals` (#2230)
+ - Fixed bug in [`ConstraintPrimal`](@ref) of [`Bridges.Constraint.QuadToSOCBridge`](@ref)
```
Comment (odow) [inline docs/src/changelog.md:26]:
```suggestion
 - Fixed bug in [`ConstraintPrimal`](@ref) of [`Bridges.Constraint.QuadtoSOCBridge`](@ref)
```

File: docs/src/changelog.md [hunk 3]
```diff
@@ -7,6 +7,31 @@ CurrentModule = MathOptInterface
...
+ - Fixed reading LP file with a `-infinity <= x <= +infinity` variable (#2225)
+ - Fixed missing `require` in `Test.test_nonlinear_duals` (#2230)
+ - Fixed bug in [`ConstraintPrimal`](@ref) of [`Bridges.Constraint.QuadtoSOCBridge`](@ref)
+   (#2240)
+
+### Other
+
+ - Added extensions to `solver-tests.yml` (#2229)
+ - Refactored `test/Benchmarks` (#2234)
+ - Fixed warngs in tests (#2241)
```
Comment (odow) [inline docs/src/changelog.md:33]:
```suggestion
 - Fixed warnings in tests (#2241)
```

File: docs/src/changelog.md [hunk 4]
```diff
@@ -7,6 +7,33 @@ CurrentModule = MathOptInterface
...
 
+## v1.19.0 (August 7, 2023)
+
+### Added
+
+ - Added [`Utilities.eval_variables`](@ref) support for
+   [`ScalarNonlinearFunction`](@ref) (#2218) (#2219)
+ - Added support for [`ScalarNonlinearFunction`](@ref) in `FileFormats.NL`
+   (#2228) (#2231)
+ - Added [`Bridges.Constraint.ScalarQuadraticToScalarNonlinearBridge`](@ref) (#2233)
```
Comment (odow) [inline docs/src/changelog.md:18]: We should revert this and then tag. Then add it back with the function-convert bridges in v1.20

**Comment by blegat** [comment]
Maybe we could add https://github.com/jump-dev/MathOptInterface.jl/pull/2237 as well, the PR is now ready

**Comment by odow** [comment]
- [ ] Requires #2235

**Comment by odow** [comment]
Updated `solver-tests`: https://github.com/jump-dev/MathOptInterface.jl/actions/runs/5862825321

**Comment by odow** [comment]
@blegat okay to merge?

**Review by blegat — APPROVED** [review]

**Pull request merged**